File: Tugas coding modul 6/coding c/PRAK605-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void matriks1(int b, int A[b][b], int B[b][b], int m[b][b]) {
    for (int i = 1; i <= b; i++) {
        for (int j = 1; j <= b; j++) {
            m[i][j] = 0;

            for (int k = 1; k <= b; k++) {
                m[i][j] += A[i][k] * B[k][j];
            }
        }
    }
}

void matriks2(int b, int matrix[b][b]) {
    for (int i = 1; i <= b; i++) {
        for (int j = 1; j <= b; j++) {
            printf("%d ", matrix[i][j]);
        }
        printf("\n");
    }
}

int main() {
    int b;

    scanf("%d", &b);

    int A[b][b], B[b][b], m[b][b];

    printf("Matriks A \n");
    for (int i = 1; i <= b; i++) {
        for (int j = 1; j <= b; j++) {
            scanf("%d", &A[i][j]);
        }
    }

    printf("Matriks B \n");
    for (int i = 1; i <= b; i++) {
        for (int j = 1; j <= b; j++) {
            scanf("%d", &B[i][j]);
        }
    }

    matriks1(b, A, B, m);

    printf("Matriks AXB \n");
    matriks2(b, m);

    return 0;
}

File: Tugas coding modul 6/coding py/PRAK605-2310817120005-Noviana_Nur_Aisyah.py
def matriks1(b, A, B):
    m = [[0] * (b + 1) for _ in range(b + 1)]

    for i in range(1, b + 1):
        for j in range(1, b + 1):
            m[i][j] = 0
            
            for k in range(1, b + 1):
                m[i][j] += A[i][k] * B[k][j]

    return m

def matriks2(b, matrix):
    for i in range(1, b + 1):
        for j in range(1, b + 1):
            print(matrix[i][j], end=" ")

        print()

b = int(input())

A = [[0] * (b + 1) for _ in range(b + 1)]
B = [[0] * (b + 1) for _ in range(b + 1)]

print("Matriks A")
for i in range(1, b + 1):
    A[i][1:b + 1] = map(int, input().split())

print("Matriks B")
for i in range(1, b + 1):
    B[i][1:b + 1] = map(int, input().split())

hasil = matriks1(b, A, B)

print("Matriks AXB")
matriks2(b, hasil)